### DISCUSSION TITLE
CTRL + ENTER no longer re-run query in new Prometheus query editor

### DISCUSSION THREAD

**Discussion by cykl** [grafana/grafana#43264] [Dashboards, Panels & Visualizations]
Query editors usually allows to re-run the query by hitting CTRL + ENTER, which is a huge time saver. 

Prometheus query editor supported this shortcut until Grafana 8.2. However, since Grafana 8.3 introduced the new monaco based query editor and CTLR + ENTER disappeared :sob: 

As it's unclear to me whether this is a deliberate choice or an oversight, I decided to create this discussion rather than filling a bug report.

1. Is there another shortcut to re-run quer(y|ies) in explore view and panel editors?
2. If not, could you re-implement CTRL + ENTER in the new Prometheus query editor?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
